data/human/deep missions.txt

```diff
@@ -3155,12 +3155,10 @@ mission "Home for Skadenga 2"
 			`	"I am Jakob Forsyth, Vice Planetary-Governor of Nifel and CEO of Nifel-Research Solutions Incorporated, and the man sitting beside me is Henrik Fogel, Deputy Lieutenant-Sub-Director of the Census and Culture Bureau for The Deep. We've been monitoring you ever since you first arrived and became entangled with those sad, backwards, hill-witches." Jakob takes a sip from an elegant glass on the table.`
 			choice
 				`	"If you've been watching me, does that mean you knew about the Ice-Crawler accident and did nothing?"`
-			`	"Of course I knew." Jakob answers. "It is my business to know. The only people who live here anymore are employees of my firm and those hill-witches." Holding up both hands in a placating gesture, he adds, "And I really am sorry we could not extract you from that avalanche. But we did a cost-benefit analysis, and it just didn't make sense. You run a ship, a business, you understand. How often do you let crew die in boarding action to make a profit?"`
-			`	Jakob puts his hands down. "Anyways, we hardly have the infrastructure to support rescue missions. Our financial team ran the numbers a few generations back, and we realized we could significantly increase our profit margins if we downsized our research division workforce to nothing, and instead focused on managing our existing patents.`
-			`	"Since then we've been shutting things down, laying people off, and sending the rest offworld. Various services, including emergency ones, are only barely functional. It's much cheaper to shut this world down rather than try and keep managing it. That's why it will be completely uninhabited within 50 years."`
+			`	"Of course I knew." Jakob answers. "It is my business to know. The only people who live here anymore are employees of my firm and those hill-witches." Holding up both hands in a placating gesture, he adds, "And I really am sorry we could not extract you from that avalanche. We don't have the infrastructure to support rescue missions anymore; various services, including emergency ones, are only barely functional. The whole operation on this planet is being shut down, you know. Not enough opportunity to make a profit left here. That's why it will be completely uninhabited within 50 years."`
```

Comment:
```suggestion
			`	"Of course I knew." Jakob answers. "It is my business to know. The only people who live here anymore are employees of my firm and those hill-witches." Holding up both hands in a placating gesture, he adds, "And I really am sorry we could not extract you from that avalanche. We don't have the resources allocated to support rescue missions anymore; various services, including emergency ones, are only barely functional. The whole operation on this planet is being shut down, you know. Not enough opportunity to make a profit left here. That's why it will be completely uninhabited within 50 years."`
```